R/range-selector.R

```diff
@@ -56,7 +56,11 @@ dyRangeSelector <- function(dygraph,
   if (!is.null(dateWindow)) {
     if (length(dateWindow) != 2)
       stop("dateWindow must be vector of length 2 that is convertible to POSIXct")
-    selector$dateWindow <- sapply(USE.NAMES = FALSE, dateWindow, asISO8601Time)
+    # TODO: Rename dateWindow argument?
```

Comment:
Yes, let's keep dateWindow then (I've always used exactly the same names as
dygraphs for the sake of consistency)

On Sun, Oct 25, 2015 at 2:10 PM Daniel Gromer <notifications@github.com>
wrote:

> In R/range-selector.R
> <https://github.com/rstudio/dygraphs/pull/91#discussion_r42947961>:
>
> > @@ -56,7 +56,11 @@ dyRangeSelector <- function(dygraph,
> >    if (!is.null(dateWindow)) {
> >      if (length(dateWindow) != 2)
> >        stop("dateWindow must be vector of length 2 that is convertible to POSIXct")
> > -    selector$dateWindow <- sapply(USE.NAMES = FALSE, dateWindow, asISO8601Time)
> > +    # TODO: Rename dateWindow argument?
>
> I just checked the dygraphs (JS) documentation and they use dateWindow
> also for numeric values (http://dygraphs.com/options.html#dateWindow).
> Not sure if we should stick with their notation because it may be easier to
> maintain dygraphs (R) when updating to a new version of dygraphs (JS).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rstudio/dygraphs/pull/91/files#r42947961>.
>